providers/manage.rb

```diff
@@ -178,3 +180,18 @@ def manage_home_files?(home_dir, user)
     true
   end
 end
+
+# This is a workaround to include recipe 'chef-vault' in LWRP.
+def chef_vault_item(bag, item)
```

Comment:
Yes, you are right, it is much cleaner: https://github.com/yl013512/users/commit/65db9e3b292a6468414dbc338041b9263ba0ae40